hetmech/test_matrix.py

```diff
@@ -71,12 +71,33 @@ def test_metaedge_to_adjacency_matrix(test_edge, dtype, threshold):
     assert (adj_mat != exp_adj).sum() == 0
 
 
-@pytest.mark.parametrize('metapath', [('CpDtCpD', 'abab'),
-                                      ('CbGeAlD', 'no_repeats'),
-                                      ('CuGiG<rGaD', 'disjoint'),
-                                      ('CuGpBPpGaD', 'disjoint'),
-                                      ('CuGuDdGuD', 'abab'),
-                                      ('CuGiGuCpD', 'abba'),
-                                      ('CuGiGr>GaD', 'disjoint')])
+@pytest.mark.parametrize('metapath', [('GiG', 'disjoint'),
+                                      ('TlDaG', 'no_repeats'),
+                                      ('DaGiGaDaG', 'other'),
+                                      ('DaGiG', 'disjoint'),
+                                      ('DaGiGaD', 'BAAB'),
+                                      ('GeTlDlTeG', 'BAAB'),
+                                      ('TlDlTlD', 'BABA'),
+                                      ('GiGiGiG', 'disjoint'),
+                                      ('TlDaGiG', 'disjoint'),
+                                      ('TlDaGeTlDaG', 0),  # ABCABC
+                                      ('DaGiGiGiGiGaD', None),  # ABBBBBA
+                                      ('GiGiGaDlTlD', 'disjoint'),
+                                      ('GiGiGiGiGiGiGiGiG', 'disjoint'),
+                                      ('DlTlDlTlD', 'other')])  # ABABA
 def test_categorize(metapath):
-    assert categorize(metapath[0]) == metapath[1]
+    url = 'https://github.com/dhimmel/hetio/raw/{}/{}'.format(
+        '9dc747b8fc4e23ef3437829ffde4d047f2e1bdde',
+        'test/data/disease-gene-example-graph.json',
```

Comment:
Just read in the metagraph `test/data/hetionet-v1.0-metagraph.json` and not the subgraph